shared/css/color.scss

```diff
@@ -59,6 +62,7 @@ $hdr_color:   $purple;
 // Some extra colors.
 $red: #c00;
 $lightest_red: #fcc;
+$redesign_red: #d5001a;
```

Comment:
I would just use our existing ones until we do the site redesign and we can choose to update to the comp colors.

Can you try these? Either update screenshots and ping me again or find me for faster iteration?

* informational - teal
* success - green
* failure - red
* warning - mustardyellow